synfig-core/src/modules/mod_ffmpeg/trgt_ffmpeg.cpp

```diff
@@ -34,8 +34,9 @@
 #	include <config.h>
 #endif
 
-#include <synfig/localization.h>
+#include <synfig/filesystemnative.h>
 #include <synfig/general.h>
+#include <synfig/localization.h>
 #include <synfig/soundprocessor.h>
 
 #include <cstdio>
```

Comment:
Looks like this includes no longer being used.

```
// MSVC
#ifndef F_OK
#define F_OK 0
#endif
```
Can be removed too.

Reply:
Done